### PR TITLE
inputview: implement autogrow for single line inputs

### PR DESCRIPTION
**Pull request by sinan**
cc/ @usirin

### CODE REVIEW

**Comment by sinan** [comment]
gif

File: lib/components/inputs/inputview.coffee [hunk 1]
```diff
@@ -449,14 +451,18 @@ module.exports = class KDInputView extends KDView
 
     # input content is copied into clone
     # element to get calculated height
-    @_clone = $ '<div/>', class : 'invisible'
+    @_clone = $ '<div/>', { class: 'invisible' }
+
+    { type } = @getOptions()
+    isVertical = type.toLowerCase() is 'textarea'
```
Comment (usirin) [inline lib/components/inputs/inputview.coffee]: Why not accepting this as an option?

Comment (sinan) [inline lib/components/inputs/inputview.coffee]: isn't it an option already?

Comment (usirin) [inline lib/components/inputs/inputview.coffee]: operation we are making is `isVertical` but the option we are passing is `textarea`. Can't an input autogrow both vertically and horizontally?

Comment (sinan) [inline lib/components/inputs/inputview.coffee]: Assumption we did earlier was to use autogrow only for textareas and just vertically, now we expand the functionality to `text` inputs basically, but they can only grow horizontally. That's why I didn't think another option was necessary.

However I didn't think about a bidirectional growing input, never needed that. Do you think it is necessary?

Comment (usirin) [inline lib/components/inputs/inputview.coffee]: Nope, I don't think so. Only the assumption we make made me confused.

**Pull request merged**